app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "../lib/oidc_client.rb"
+
+class SessionsController < ApplicationController
+  def create
+    if logged_in?
+      redirect_to default_redirect_path
+      return
+    end
+
+    if Rails.env.test?
+      render plain: "Redirecting to login"
+    else
+      redirect_to oidc.auth_uri(return_path: params["redirect_path"])[:uri]
```

Comment:
Could you also call this parameter `redirect_path`, to be consistent?

Reply:
Done